packages/admin-ui/ui/src/domain/orders/details/order-line/index.tsx

```diff
@@ -99,6 +103,23 @@ const ReservationIndicator = ({
 
   const awaitingAllocation = allocatableSum - reservationsSum
 
+  if (!lineItem.variant_id || (!isLoading && !variant?.inventory.length)) {
+    return (
+      <div className="text-grey-40">
+        <Tooltip
+          content={
+            <div className="inter-small-regular flex flex-col items-center px-1 pt-1 pb-2">
+              No inventory item exist for this variant.
```

Comment:
thought: I am not sure this is super intuitive from a merchant perspective, as inventory items could be interpreted as "standard inventory" as well. 

That said, I don't have a good suggestion other than removing the badge entirely, when we are dealing with items without dedicated inventory items.

Wdyt?

<img width="1468" alt="CleanShot 2023-03-31 at 13 48 44@2x" src="https://user-images.githubusercontent.com/59018053/229112511-073df630-6ad3-4ba1-894f-72acf8677c05.png">

Reply:
While seeing your comment, i think it would make sense not displaying the badge when no inventory dedicated to avoid confusion +1

Reply:
👍